apps/explorer/lib/explorer/chain/events/db_sender.ex

```diff
@@ -21,6 +25,8 @@ defmodule Explorer.Chain.Events.DBSender do
   end
 
   defp send_notify(payload) do
-    Repo.query!("select pg_notify('chain_event', $1::text);", [payload])
+    if byte_size(payload) < @max_payload do
+      Repo.query!("select pg_notify('chain_event', $1::text);", [payload])
+    end
```

Comment:
We could add an if statement + log here to get some visibility when a contract has not been sent to the db due to exceeding byte size